test/k8sT/Nightly.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2017 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package k8sTest
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/cilium/cilium/test/helpers"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	log "github.com/sirupsen/logrus"
+)
+
+var _ = Describe("K8sNightlyTest", func() {
```

Comment:
Change this to be "NightlyK8sEpsMeasurement" - that way we can run ginkgo commands with a specific focus, i.e. `ginkgo --focus="NightlyK8s*" -v -noColor` for K8s nightly tests. Otherwise, this will get run with all Ginkgo tests since it currently is prefixed with "K8s"

Alternatively, we could have a folder called nightly and throw all nightly tests in there?

Reply:
Done